### PR TITLE
Viresh_Assignment_9_Done

### PR DESCRIPTION
**Pull request by VireshJJun20**
Please review and comment.

### CODE REVIEW

File: src/vireshJJun20/ArithOps.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+	double sum(double a, double b, double c){
+		double smAnswer;
+		smAnswer= a+b+c;
+		System.out.println(" - Sum: " + smAnswer);
+		return smAnswer;
+	}
+	
+	double average(double a, double b, double c){
+		ArithOps ar= new ArithOps();
```
Comment (KrishnaKTechnocredits) [inline src/vireshJJun20/ArithOps.java]: No need to create new object. non static to non static method can be called directly.

File: src/vireshJJun20/ArithOps.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+		System.out.println(" - Average: " +avAnswer);
+		return avAnswer;
+		}
+	}
+	
+	static boolean isAnswerAboveExpecation(double avAnswer){
+		boolean boolExp = false;
+			if (avAnswer> 50){
+				boolExp= true;
+
```
Comment (KrishnaKTechnocredits) [inline src/vireshJJun20/ArithOps.java]: remove extra space

File: src/vireshJJun20/ArithOps.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+	public static void main(String[] args){
+		ArithOps arithOps= new ArithOps();
+		double avg= arithOps.average(99.0,99.0,99.0);
+		if (avg!=00.0) {
+			if(ArithOps.isEligible(avg)== true) {
+				System.out.println("Is Eligible: Yes (Average is more than 50 or divisible by 2)");
+			}
+			else
+				System.out.println("Is Eligible: No (Average is less than or equals 50 or not divisible by 2)");
+				//System.out.println("Is Eligible: " + ArithOps.isEligible(avg));
```
Comment (KrishnaKTechnocredits) [inline src/vireshJJun20/ArithOps.java]: It should be and.
if not divisible by 2 and less than or equals 50...
System.out.println("Is Eligible: No (Average is less than or equals 50 or not divisible by 2)");

File: src/vireshJJun20/ArithOps.java [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+		else{
+			grade= "Grade B";
+		}
+		return grade;
+	}	
+	
+	public static void main(String[] args){
+		ArithOps arithOps= new ArithOps();
+		double avg= arithOps.average(99.0,99.0,99.0);
+		if (avg!=00.0) {
```
Comment (KrishnaKTechnocredits) [inline src/vireshJJun20/ArithOps.java]: If avg is == 00.0 then we dont want to even call other methods like to getYourGrade or isEligible ?

Comment (VireshJJun20) [reply src/vireshJJun20/ArithOps.java]: Done with the changes

File: src/vireshJJun20/ArithOps.java (hunk 3 above)
Comment (VireshJJun20) [reply src/vireshJJun20/ArithOps.java]: Done with the changes.

File: src/vireshJJun20/ArithOps.java (hunk 2 above)
Comment (VireshJJun20) [reply src/vireshJJun20/ArithOps.java]: Done.

File: src/vireshJJun20/ArithOps.java (hunk 1 above)
Comment (VireshJJun20) [reply src/vireshJJun20/ArithOps.java]: Removed the step 13 (object creation with ref variable ar)

**Pull request merged**